web_tree_image_tooltip/view/assets.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
```

Comment:
License/Copyright is option, yet appreciated

Reply:
> License/Copyright is option, yet appreciated

It's Not required